dts/bindings/sensor/st,lps22hh-common.yaml

```diff
@@ -18,9 +18,9 @@ properties:
       default: 0
       description: |
           Specify the default output data rate expressed in samples per second (Hz).
-          Default is power-down mode
+          The default of 0 is the power-on reset value.
       enum:
-        - 0  # Power-Down
+        - 0  # Power-Down (default)
```

Comment:
It's redundant to say "The default of 0" in the description and "Power-Down (default)" in the enum. Let's keep a single source of truth for the default value like this:
```
      default: 0
      description: |
          Specify the default output data rate expressed in samples per second (Hz).
          The default is the power-on reset value.
      enum:
        - 0  # Power-Down
```

Reply:
ok, ack